tests/nist/nist_test.sh

```diff
@@ -21,6 +21,65 @@ function test_nist {
 	fi
 }
 
+function test_results_directive {
+	test_dir="$1"
+	run_script="$(cd "${builddir}"; pwd)/run"
```

Comment:
Yes, I noticed that, but wanted to keep it consistent with `test_nist()`, which also uses `${run_script}`.